webpack.config.js

```diff
@@ -0,0 +1,29 @@
+var path = require('path');
```

Comment:
Since we are using ES6 it would be best practices to use `const` or `let` in place of `var` unless you have a specific scoping reason not to. I'd recommend `const` here :)